raiden-ts/tests/integration/provider.ts

```diff
@@ -10,14 +10,17 @@ import { parseUnits } from '@ethersproject/units';
 
 import { ContractsInfo } from 'raiden-ts/types';
 import { Address, last } from 'raiden-ts/utils/types';
-import { TokenNetworkRegistry } from 'raiden-ts/contracts/TokenNetworkRegistry';
-import { CustomToken } from 'raiden-ts/contracts/CustomToken';
-import { ServiceRegistry } from 'raiden-ts/contracts/ServiceRegistry';
-import { TokenNetworkRegistryFactory } from 'raiden-ts/contracts/TokenNetworkRegistryFactory';
-import { UserDeposit } from 'raiden-ts/contracts/UserDeposit';
-import { SecretRegistry } from 'raiden-ts/contracts/SecretRegistry';
-import { MonitoringService } from 'raiden-ts/contracts/MonitoringService';
-import { OneToN } from 'raiden-ts/contracts/OneToN';
+import type {
+  TokenNetworkRegistry,
+  CustomToken,
+  ServiceRegistry,
+  UserDeposit,
+  SecretRegistry,
+  MonitoringService,
+  OneToN,
+} from 'raiden-ts/contracts';
+
+import { TokenNetworkRegistry__factory } from 'raiden-ts/contracts/';
```

Comment:
You should merge this one with above. As there'll be a factory between imports, you can't use `import type` then. Don't leave trailing slash.